submissions/kriLEBED/html_css_popup/index.html

```diff
@@ -0,0 +1,149 @@
+                                        <img src="img/files-icon.png" alt="files">
+                                        <p>Files</p>
+                                    </a>
+                                </li>
+                                <li class="submenu-item">
+                                    <a class="submenu-link" href="#">
+                                        <img src="img/contacts-icon.png" alt="contacts">
+                                        <p>Contacts</p>
+                                    </a>
+                                </li>
+                                <li class="submenu-item">
+                                    <a class="submenu-link" href="#">
+                                        <img src="img/translate-icon.png" alt="translate">
+                                        <p>Translate</p>
+                                    </a>
+                                </li>
+                                <li class="submenu-item">
+                                    <a class="submenu-link" href="#">
+                                        <img src="img/videos-icon.png" alt="videos">
+                                        <p>Videos</p>
+                                    </a>
+                                </li>
+                                <li class="submenu-item">
+                                    <a class="submenu-link" href="#">
+                                        <img src="img/search-icon.png" alt="search">
+                                        <p>Search</p>
+                                    </a>
+                                </li>
+                                <li class="submenu-item">
+                                    <a class="submenu-link" href="#">
+                                        <img src="img/photo-icon.png" alt="photo">
+                                        <p>Photo</p>
+                                    </a>
+                                </li>
+                                <li class="submenu-item">
+                                    <a class="submenu-link" href="#">
+                                        <img src="img/music-icon.png" alt="music">
+                                        <p>Music</p>
+                                    </a>
+                                </li>
+                                <li class="submenu-item">
+                                    <a class="submenu-link" href="#">
+                                        <img src="img/document-icon.png" alt="document">
+                                        <p>Document</p>
+                                    </a>
+                                </li>
+                            </ul>
+                            <div class="more_button">
+                                <label for="more"></label>
+                                <input type="checkbox" id="more" class="more_btn"/>
```

Comment:
All single tags should be without slashes ("/") due to HTML5 specification. Could you please fix it?